doc/source/fluid_properties/fluids.rst

```diff
@@ -4,8 +4,9 @@ Fluid Properties in pandapipes
 
 The pipelines in the pandapipes network are run with a certain fluid. This can be
 chosen individually for the net from the fluid library or by own creation. The fluids
-are defined by certain properties. In the fluid library currently high or low caloric natural
-gas (hgas and lgas), water and air are implemented with default properties.
+are defined by certain properties. In the fluid library currently high and low caloric natural
+gas (hgas and lgas), water, air, carbon dioxide, ethane, hydrogen, methane, nitrogen and oxygen
```

Comment:
carbon dioxide, ethane, hydrogen, methane, nitrogen and oxygen are available in the properties folder, but not yet implemented in call_lib. So I suggest to not mention it in the documentation at this point.
```suggestion
gas (hgas and lgas), water and air.
```